### DISCUSSION TITLE
Blog Post About v5

### DISCUSSION THREAD

**Discussion by ghost** [phalcon/phalcon#109] [General]
There are few errors in this post @ https://blog.phalcon.io/post/phalcon-v5-0-0-alpha-1-released

`https://github.com/orgs/phalcon/projects/4`

I think it should be;

`https://github.com/orgs/phalcon/projects/5`

Also why you talking about DLL for windows?
`We are still working on the Windows DLLs. For our friends using Windows, please be a bit patient. They will be introduced in subsequent versions of v5.`

v5 was coded in dymaic php right?

**Comment by niden** [comment]
We have found some misalignments to interfaces (still). Therefore in order to follow SemVer, we have released v5 Aplha which will be again a PHP extension. v6 will be the native PHP one

https://blog.phalcon.io/post/community-hangout-status-update-2021-february
